rollup.config.js

```diff
@@ -58,37 +58,60 @@ const customBabelConfig = {
   exclude: /node_modules/,
 };
 
-export default {
-  input: './index.js',
-  output: {
-    file: './build/bundle.js',
-    name: 'blib',
-    format: 'umd',
-    globals: {
-      react: 'React',
-      'prop-types': 'PropTypes',
-      axios: 'axios',
-      'react-dates': 'reactDates',
-      'react-selectize': 'reactSelectize',
-      'google-map-react': 'GoogleMap',
-      'google-map-react/utils': 'utils',
-      'lodash/sortBy': 'sortBy',
-      'react-pdf-js': 'PDF',
+export default [
+  {
+    input: './src/functions.js',
+    output: {
+      file: './build/functions.js',
+      name: 'blib',
+      format: 'umd',
+      globals: {
+        react: 'React',
+        'prop-types': 'PropTypes',
```

Comment:
should be able to remove most of these globals, see what the build output moans about